examples/air-quality-sensor-app/air-quality-sensor-common/src/AirQualitySensorAttrUpdateHandler.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include <platform/PlatformManager.h>
+
+#include "AirQualitySensorAttrUpdateHandler.h"
+#include <air-quality-sensor-manager.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::AirQuality;
+using namespace chip::DeviceLayer;
+
+AirQualitySensorAttrUpdateHandler * AirQualitySensorAttrUpdateHandler::FromJSON(const char * json)
+{
+    Json::Reader reader;
+    Json::Value value;
+
+    if (!reader.parse(json, value))
+    {
+        ChipLogError(NotSpecified,
+                     "AirQualitySensor App: Error parsing JSON with error %s:", reader.getFormattedErrorMessages().c_str());
+        return nullptr;
+    }
+
+    if (value.empty() || !value.isObject())
+    {
+        ChipLogError(NotSpecified, "AirQualitySensor App: Invalid JSON command received");
+        return nullptr;
+    }
+
+    if (!value.isMember("Name") || !value["Name"].isString())
+    {
+        ChipLogError(NotSpecified, "AirQualitySensor App: Invalid JSON command received: command name is missing");
+        return nullptr;
+    }
+
+    return Platform::New<AirQualitySensorAttrUpdateHandler>(std::move(value));
+}
+
+void AirQualitySensorAttrUpdateHandler::HandleCommand(intptr_t context)
+{
+    auto * self             = reinterpret_cast<AirQualitySensorAttrUpdateHandler *>(context);
+    std::string clusterName = self->mJsonValue["Name"].asString();
```

Comment:
Do we want to place common code that uses std::string and json? 

Concern is that we tried to avoid std::string/list/map and other things using heap from being used in embedded and examples end up being copied & pasted as real products, hence my concern of using JSON and stl here. It seemed fine in the original linux side because linux/darwin/android are large systems and they are ok with such things, however embedded seems more sensitive.